From 59fc931ac50627eafe267fe949c9fc0c0018918a Mon Sep 17 00:00:00 2001
From: Jose Luis Blanco-Claraco <joseluisblancoc@gmail.com>
Date: Wed, 14 Jun 2023 00:05:40 +0200
Subject: [PATCH] debian/rules: use GTSAM_USE_SYSTEM_EIGEN=ON

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index 284a071..a7c2535 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -37,6 +37,7 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DGTSAM_USE_SYSTEM_EIGEN=ON \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.34.1

